### DISCUSSION TITLE
Confunde mensaje: Existe una versión más reciente para este bot

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#230] [2. Questions and Answers] [answered]
Hola

Qué significa ese mensaje?:

Fundamento: Estaba modificando un bot y en las rutinas encuentro ese mensaje, veo las versiones y existe una 2.0.0. En el momento de seleccionar, me aparece vacío todos los campos y cuando quiero retomar la anterior se me borra.

Qué debe pasar realmente, ya que el mensaje confunde (por lo menos a mí)

Adjunto imagenes de manera secuencial
1.- ![image](https://user-images.githubusercontent.com/61419471/120718293-1799e700-c497-11eb-8248-92d9af415960.png)

2.- ![image](https://user-images.githubusercontent.com/61419471/120718333-241e3f80-c497-11eb-9704-d3f82fd8b280.png)

3.- ![image](https://user-images.githubusercontent.com/61419471/120718393-2ed8d480-c497-11eb-843b-48c2ad2f436c.png)

4.- ![image](https://user-images.githubusercontent.com/61419471/120718402-339d8880-c497-11eb-95a2-57e531c0f734.png)

O quizás debe pasar:

1.- En el momento que edite el bot, se me guarde como una versión dos y allí puedo escoger entre una u otra versión?.

Gracias

**Comment by eduborgono** [comment, marked as answer]
@NicolasPLG este mensaje hace refercia a que la etapa del bot tiene una segunda versión. Se refiere al PBBot de la etapa y no a la rutina del bot en general.
Las entradas se borran porque pueden haber cambiado los parametros del bot, haciendo algunos incompatibles con los de la nueva versión.

**Reply by NicolasPLG** [reply]
gracias @eduborgono
